requirements.txt

```diff
@@ -2,3 +2,4 @@
 pyyaml
 jinja2
 tabulate
+graphviz
```

Comment:
```suggestion
graphviz
```
Something in your editor settings seems to strip off the last newline in each file.

Reply:
Yes, you're right. I'll check for the new lines from next time. Thanks!